src/NzbDrone.Core/Download/CompletedDownloadService.cs

```diff
@@ -96,6 +96,17 @@ public void Check(TrackedDownload trackedDownload)
                 if (series == null)
                 {
                     trackedDownload.Warn("Series title mismatch; automatic import is not possible. Check the download troubleshooting entry on the wiki for common causes.");
+
+                    if (!trackedDownload.HasNotifiedManualInteractionRequired)
```

Comment:
you mean in this case don't notify if it was an interactive search, with the expectation that user is already present and ready to import?

Reply:
I'm not sure I agree here. I this case it would stop all warnings when the title was a mismatch. I didn't change the logic here of the warning creation, only added the notifcation